src/chainparams.h

```diff
@@ -24,6 +23,14 @@ struct SeedSpec6 {
     uint16_t port;
 };
 
+typedef std::map<int, uint256> MapCheckpoints;
```

Comment:
It is true that it doesn't add much readability at this point. Maybe you can fix it with a commit to the trivial branch?